src/components/ResourcesSection/index.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Col, Row } from "antd";
+import "./index.css";
+import "antd/dist/antd.css";
+import { Link, BrowserRouter as Router } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import recycleImg from "./../../images/recycleResourceSetion.svg";
+import earthImg from "./../../images/earthResourceSetion.svg";
+import pollutionImg from "./../../images/pollutionResourceSetion.svg";
+
+const ResourcesSection = () => {
+  const [t] = useTranslation();
+  return (
+    // "site-card-wrapper" is a built in ant Designe className
+    <div className="site-card-wrapper">
```

Comment:
this feels wrong (you should never have to use an antd class like this). the comment is helpful but what is it for? antd components should be used like you use row and col

Reply:
I've inserted this comment to justify why I'm using "antd className" and not and not the "BME". and for the usage of the antd class, I didn't understand the issue. I've used this since the first day I did this PR and you didn't guide me about any issue using it like that. I've updated the PR now addressing the other feedback. Thanks.

Reply:
just because i didn't notice it on the first day doesn't mean i can't notice it later :) given how many PRs 

can you show me the documentation of where this class is from antd? are you sure its a built in antd class?

when we use antd components, we do it like this:
```
<Card>
</Card>
```

we never do it like this:
```
<div className="site-card-wrapper">
```
and i don't see anything on google related to this

can you explain to me in words what this class is supposed to be doing?

Reply:
@louisrli 😄, I totally agree about the amount of PR you have to review, Thank you!!. 
Kindly see this link : 
https://ant.design/components/card/ 
check: "Card in column"

Reply:
thanks. i can see how the demo would be confusing, but actually there is nothing special about that class name. it's used for the demo

css rules
![image](https://user-images.githubusercontent.com/1303401/99260400-3a262480-2813-11eb-8d47-383ecbd98a2e.png)

![image](https://user-images.githubusercontent.com/1303401/99260419-40b49c00-2813-11eb-90ea-1125b8bda04b.png)


what their demo looks like. this css rule is what gives their demo the grey background or whatever

so basically, you can change this div to be resourceCardWrapper (camelcase), delete the antd comment, update your css class names appropriately. your code will still work exactly the same.

Reply:
well noted. will update as you requested. Thanks

Reply:
PTAL